Cargo.toml

```diff
@@ -13,7 +13,116 @@ categories = ["command-line-interface"]
 include = ["src/**/*", "README.md"]
 
 [dependencies]
-maplit = "1.0.2"
-strum = { version = "0.24.0", features = ["derive"] }
 once_cell = "1.13.0"
-colored = "2.0.0"
\ No newline at end of file
+colored = "2.0.0"
+paste = "1.0.11"
+
+[features]
+default = ["all"]
```

Comment:
I don't think the default should be set to `all`, since it would kind of undermine the whole point of using features. I would probably set it as `dots` since that seems to be the most commonly used spinner variant.

Reply:
Changed default feature to `dots` in 41e21ed8f1f51803cb9325d8718b3d6bfeedc7ee